hugegraph-core/src/main/java/com/baidu/hugegraph/backend/tx/GraphIndexTransaction.java

```diff
@@ -800,14 +800,14 @@ private MatchedIndex collectMatchedIndex(SchemaLabel schemaLabel,
         Set<IndexLabel> ils = InsertionOrderUtil.newSet();
         for (Id il : schemaLabel.indexLabels()) {
             IndexLabel indexLabel = schema.getIndexLabel(il);
-            if (indexLabel.indexType().isUnique()) {
+            if (indexLabel == null || indexLabel.indexType().isUnique()) {
                 continue;
             }
             ils.add(indexLabel);
         }
         if (this.graph().readMode().showOlap()) {
             for (IndexLabel il : schema.getIndexLabels()) {
-                if (il.olap()) {
+                if (il != null && il.olap()) {
```

Comment:
seems getIndexLabels() won't return null element

Reply:
![image](https://user-images.githubusercontent.com/24729637/140516113-99aad64c-be83-48eb-b8bc-c3b95cd01143.png)
seems may return null element

Reply:
![image](https://user-images.githubusercontent.com/24729637/140516113-99aad64c-be83-48eb-b8bc-c3b95cd01143.png)
seems may return null element

Reply:
can fix it in getAllSchema() and skip null entry